### DISCUSSION TITLE
The child processes do not get killed after closing the SSH window

### DISCUSSION THREAD

**Discussion by ElyesBH-hub** [PowerShell/Win32-OpenSSH#2240] [General]
Hello,

I have the same issue with my OpenSSH_for_Windows_9.5p1, LibreSSL 3.8.2, deployed on Windows Server 2022. I run some scripts from a client, and I run from the client: ssh user@serverssh scp ........

When the user closes the SSH connection, the child processes remain running on the server. How can I configure it so that the child processes are killed when the SSH process is killed?

configuration is 
PasswordAuthentication no
ChallengeResponseAuthentication no
PubkeyAuthentication yes
TCPKeepAlive yes
KeepAlive yes
ClientAliveInterval 120

**Comment by maertendMSFT** [comment]
Are you creating the new processes? Or are they being created automatically? What are your scripts running?

For processes on the host, if it is created automatically, then the process should terminate. If you created the job yourself or via your script, then the process should persist.
